tutorials/vbpr_text.ipynb

```diff
@@ -0,0 +1,215 @@
+    "features = item_text_modality.count_matrix.A\n",
+    "item_image_modality = ImageModality(features=features, ids=movie_ids)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In Cornac, image recommendation models always work with `ImageModality`, similarly for other modalities. That design makes it consistent with models' original assumptions, thus it's easy to integrate models into the library without any confusion of which modality to use.\n",
+    "\n",
+    "## Experiment\n",
+    "\n",
+    "We employ `RatioSplit` evaluation method to split the rating data. The `item_image_modality` is also supplied here for later usage of model training."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ratio_split = RatioSplit(data=ml_100k, test_size=0.9,\n",
+    "                         item_image=item_image_modality,\n",
+    "                         exclude_unknowns=True, \n",
+    "                         verbose=True, seed=123)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "VBPR model is supported inside Cornac. Note that the implementation of VBPR uses [PyTorch](https://pytorch.org/), thus we need to install that dependency in order to run the model (done at the beginning). List of implemented models with their additional dependencies is carefully documented [here](https://github.com/PreferredAI/cornac#models).\n",
+    "We also use [BPR](https://arxiv.org/ftp/arxiv/papers/1205/1205.2618.pdf) model as a baseline to see the effectiveness of the text auxiliary data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vbpr = cornac.models.VBPR(k=10, k2=10, n_epochs=20, batch_size=20, learning_rate=0.001,\n",
+    "                          lambda_w=1.0, lambda_b=0.0, lambda_e=100.0, use_gpu=True, seed=123)\n",
+    "\n",
+    "bpr = cornac.models.BPR(k=20, max_iter=100, learning_rate=0.001, lambda_reg=0.001, seed=123)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "auc = cornac.metrics.AUC()\n",
+    "rec_50 = cornac.metrics.Recall(k=50)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cornac.Experiment(eval_method=ratio_split,\n",
+    "                  models=[bpr, vbpr],\n",
+    "                  metrics=[auc, rec_50]).run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Results after running the experiment:\n",
+    "\n",
+    "<pre>\n",
+    "     |    AUC | Recall@50 | Train (s) | Test (s)\n",
+    "---- + ------ + --------- + --------- + --------\n",
+    "BPR  | 0.7355 |    0.2297 |    1.3699 |  12.9997\n",
+    "VBPR | 0.7669 |    0.2499 |   61.1777 |  17.2591\n",
+    "</pre>"
```

Comment:
*Suggestion:* say some words about the results and how they support cross-modality utilization of existing models.

Reply:
noted